src/vm/reflectioninvocation.cpp

```diff
@@ -2857,6 +2862,7 @@ FCIMPL3(void, ReflectionInvocation::ExecuteCodeWithGuaranteedCleanup, Object* co
         // attempt to run the backout code.
         PrepareMethodDesc(g_pExecuteBackoutCodeHelperMethod, Instantiation(), Instantiation(), FALSE, TRUE);
     }
+#endif // FEATURE_CER
 
     ExecuteCodeWithGuaranteedCleanupHelper(&gc);
```

Comment:
Yes, I can make a more targeted fix for #8148, similar to what we did for dotnet/corefx#12412. I'll get it done this morning.